libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/dh-processes-detail-item/dh-processes-detail-item.component.html

```diff
@@ -0,0 +1,48 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div class="child-row state-success">
+  <div class="child-cell">{{ detail.name }}</div>
+  <div class="child-cell">
+    <ng-container *ngIf="detail.createdDate">
+      <watt-icon
+        name="date"
+        [size]="iconSize.Small"
+        class="date-icon"
+      ></watt-icon>
+      <span class="watt-text-s">{{ detail.createdDate | dhDateTime }}</span>
+    </ng-container>
+  </div>
+  <div class="child-cell">
+    <ng-container *ngIf="detail.effectiveDate">
+      <watt-icon
+        name="date"
+        [size]="iconSize.Small"
+        class="date-icon"
+      ></watt-icon>
+      <span class="watt-text-s">{{ detail.effectiveDate | dhDate }}</span>
+    </ng-container>
+  </div>
+  <div class="child-cell details-status-cell">{{ detail.status }}</div>
```

Comment:
I had a talk with our PO about it and this is the v1 solution. We need to figure out a better way to handle these texts from our backend, since we need the same strings in our backend when sending out XML messages.
We asked for some input here but we didn't reach any conclusions yet https://teams.microsoft.com/l/message/19:91c8a12a0ed34734b72b5678596377f5@thread.tacv2/1644844184349?tenantId=f7619355-6c67-4100-9a78-1847f30742e2&groupId=e1b7877a-ee7d-4505-a4b2-4989abc2d22a&parentMessageId=1644844184349&teamName=GreenForce&channelName=Developers&createdTime=1644844184349

Reply:
Well, if the string is needed in the backend as well, we can still use it in the frontend but instead of displaying it directly, we can map it to a translation key and we show that instead. This can be done in a future iteration/version.